test/unit/header-list.test.js

```diff
@@ -1,43 +1,43 @@
-var expect = require('expect.js'),
+var expect = require('chai').expect,
     sdk = require('../../lib/index.js'),
 
     HeaderList = sdk.HeaderList;
 
 describe('HeaderList', function () {
     it('should be a working constructor', function () {
-        expect(new HeaderList()).be.a(HeaderList);
+        expect(new HeaderList()).to.be.an.instanceOf(HeaderList);
     });
 
     it('should be able to initialise with header as string', function () {
         var hl = new HeaderList(null, 'Accept: *\nContent-Type: text/html');
-        expect(hl.count()).eql(2);
+        expect(hl.count()).to.equal(2);
     });
 
     it('should be able to export headers to string', function () {
         var hl = new HeaderList(null, 'Accept: *\nContent-Type: text/html');
-        expect(hl.toString()).eql('Accept: *\nContent-Type: text/html');
+        expect(hl.toString()).to.eql('Accept: *\nContent-Type: text/html');
```

Comment:
Use `.eql` only as a shorthand for deep comparisons, the one here (and the one below) should use `.equal` instead.